### PR TITLE
Centralize brooks jeffrey tests

### PR DESCRIPTION
**Pull request by terinpw**
#675

### CODE REVIEW

File: recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py [hunk 1]
```diff
@@ -35,35 +38,37 @@
...
-class TestBrooksJeffreyScraper(BaseScraperTest, unittest.TestCase):
+class BrooksJeffreyScraperTest(BaseScraperTest):
     """Test class for TestBrooksJeffreyScraper."""
 
-    def _init_scraper_and_yaml(self):
-        self.scraper = BrooksJeffreyScraper('us_ar_van_buren')
-        self.yaml = self.scraper.mapping_filepath
-
+    def _get_base_url(self):
+        # return https://www.vbcso.com/'
```
Comment (jamwalla) [inline recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py]: stray comment? I also dont get the rpartition thing but if its necessary all good

File: recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py [hunk 2]
```diff
@@ -88,4 +93,15 @@ def test_populate_data(self):
...
         arrest.agency = "Agency"
 
-        self.validate_and_return_populate_data(_PERSON_PAGE_HTML, expected_info)
+        self.validate_and_return_populate_data(
+            _get_person_page(), expected_info)
+
+
+def _get_person_page():
+    # Make defensive copy since content is mutable
+    return copy(_PERSON_PAGE_HTML)
```
Comment (jamwalla) [inline recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py]: wait, like a scraper does this?! we made sure dataextractor makes a copy

Comment (terinpw) [reply recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py]: Chatted on Friday - but the brooks jeffrey scraper does content.make_links_absolute...

File: recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py (hunk 1 above)
Comment (terinpw) [reply recidiviz/tests/ingest/vendors/brooks_jeffrey/brooks_jeffrey_scraper_test.py]: Removed

**Pull request merged**